ui/src/pages/insightDetail/components/eventAggregator/index.tsx

```diff
@@ -67,6 +67,7 @@ const EventAggregator: React.FC<EventAggregatorProps> = ({
   const eventSource = useRef<EventSource>()
```

Comment:
The pull request title and description are not properly set and appear to be empty. Could you please provide a meaningful title and description that explains the purpose of this change?